docs/pages/guides/authentication.md

```diff
@@ -2330,7 +2330,7 @@ exp://u.expo.dev/F767ADF57-B487-4D8F-9522-85549C39F43F?channel-name=main&runtime
 
 - **Environment:** Production projects that you `eas update`'d and opened in the Expo Go app.
 - **Create:** Use `AuthSession.makeRedirectUri({ useProxy: false })` to create this URI.
-  - The link is constructed from your Expo username and the Expo app name, which are appended to the Expo Go app URI scheme.
+  - The link is constructed from your Expo username and the app's EAS Update `slug`, which are appended to the Expo Go app URI scheme.
```

Comment:
'slug' isn't specific to EAS Update, in fact the opposite.

Reply:
```suggestion
  - The link is constructed from your Expo username and the Expo config (**app.json**) `slug`, which are appended to the Expo Go app URI scheme.
```